nntrainer/models/model_loader.cpp

```diff
@@ -279,6 +279,12 @@ int ModelLoader::loadBackboneConfigIni(dictionary *ini,
     graph[0]->setProperty(Layer::PropertyType::input_shape, input_shape);
```

Comment:
I am not sure here but is graph[0] still valid here?

Reply:
getGraph is just get the layers which generated by model loader. It is not the model_grap.Sorted. So graph[0] is still valid. and also we need to consider that the first layer of ini might be the first of layers. ( I guess we need to change )

Reply:
yeah, lets add a TODO here.

Reply:
done